_delphi_utils_python/tests/test_smooth.py

```diff
@@ -167,11 +167,10 @@ def test_impute(self):
         with pytest.raises(ValueError):
             imputed_signal = smoother.savgol_impute(signal)
 
-        # test window_length > len(signal)
+        # test window_length > len(signal) and boundary_method="identity"
         signal = np.arange(20)
         smoother = Smoother(smoother_name="savgol", boundary_method="identity", window_length=30)
-        with pytest.raises(ValueError):
-            smoothed_signal = smoother.smooth(signal)
+        smoothed_signal = smoother.smooth(signal)
```

Comment:
should there be some assertion on the output of this? I know it used to be a value error and now it doesn't, so checking it returned right would further verify behavior

Reply:
+1, @dshemetov ?

Reply:
Good catch, added an assertion 👍